mne/simulation/source.py

```diff
@@ -218,3 +220,126 @@ def simulate_stc(src, labels, stc_data, tmin, tstep, value_fun=None):
 
     stc = SourceEstimate(data, vertices=vertno, tmin=tmin, tstep=tstep)
     return stc
+
+
+def simulate_stc_from_dipole(dipole, bem, info, trans=None, min_dist=5.0,
```

Comment:
Agreed... Does this really belong under `simulation`? I get your previous comment about keeping `Dipole` a simple container, but what about a module-level method in `dipole.py`? Import like this makes sense to me:

```
from mne.dipole import make_forward_dipole
```

Almost ready with tests, will push when we've agreed on placement and name.